### PR TITLE
Feature/task 3

### PR DESCRIPTION
**Pull request by Neytus**

### CODE REVIEW

File: onboarding-task/test/reducers/listReducers/nodesInfoReducer.test.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { OrderedMap } from 'immutable';
+
+import * as actions from '../../../src/actions/actionCreators';
+import { NodeInfo } from '../../../src/models/NodeInfo';
+import { nodesInfoReducer } from '../../../src/reducers/listReducers/nodesInfoReducer';
+import { addNodeFactory } from '../../../src/actions/addNodeFactory';
+
+describe('nodesInfoReducer', () => {
+
```
Comment (Suzii) [inline onboarding-task/test/reducers/listReducers/nodesInfoReducer.test.js]: Redundant newline

File: onboarding-task/test/reducers/listReducers/nodesInfoReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      const actualState = nodesInfoReducer(nonEmptyState, action);
+
+      expect(actualState).toEqual(emptyState);
+    });
+
+    it('handles deleting a nonexistent node', () => {
+      const action = actions.deleteNode(id);
+
+      const actualState = nodesInfoReducer(emptyState, action);
```
Comment (Suzii) [inline onboarding-task/test/reducers/listReducers/nodesInfoReducer.test.js]: I would say you should also test deleting non-existing node if the sate is not empty.

Comment (Suzii) [inline onboarding-task/test/reducers/listReducers/nodesInfoReducer.test.js]: Same goes for this king of test in other reducers.

File: onboarding-task/src/reducers/listReducers/nodesIdsReducer.js [hunk 3]
```diff
@@ -9,8 +9,13 @@ export const nodesIdsReducer = (state = List(), action) => {
   switch (action.type) {
     case ADD_NODE:
       return state.push(action.payload.id);
-    case DELETE_NODE:
-      return state.delete(state.indexOf(action.payload.id));
+    case DELETE_NODE: {
```
Comment (Suzii) [inline onboarding-task/src/reducers/listReducers/nodesIdsReducer.js]: :+1:
